### PR TITLE
init at 2.0.0 Fuchsia Pop

### PR DESCRIPTION
**Pull request by transgirllucy**
###### Description of changes

added Fuchsia Pop cursor theme

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -10282,6 +10282,12 @@
     github = "MrTarantoga";
     githubId = 53876219;
   };
+  mrtuxa = {
+    email = "mrtuxa@leipzig.freifunk.net";
+    name = "mrtuxa";
+    github = "mrtuxa";
+    githubId = 69870860;
+  }
```
Comment (Zaechus) [inline maintainers/maintainer-list.nix:10290]:
```suggestion
  };
```
Forgot a semicolon

File: pkgs/desktops/cursor/fuchsia-pop/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  '';
+
+  meta = with lib; {
+    description = "Fuchsia cursor Theme";
+    homepage = "https://github.com/ful1e5/fuchsia-cursor";
+    license = licenses.gpl3;
+    platforms = platforms.all;
+    maintainers = [ mataintainers.mrtuxa ];
+  };
+}
```
Comment (Zaechus) [inline pkgs/desktops/cursor/fuchsia-pop/default.nix:33]:
```suggestion
}

```
No newline at end of file

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -39559,4 +39559,5 @@ with pkgs;
   udict = callPackage ../applications/misc/udict { };
 
   duden = callPackage ../applications/misc/duden { };
+  fuchsia-pop = callPackage ../cursor/fuchsia-pop
```
Comment (Zaechus) [inline pkgs/top-level/all-packages.nix:39562]:
```suggestion

  fuchsia-pop = callPackage ../cursor/fuchsia-pop
```

File: pkgs/desktops/cursor/fuchsia-pop/default.nix [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  noDumpEnvVars = true;
+
+  sourceRoot = ".";
+
+  installPhase = ''
+  runHook preInstall
+  mkdir -p $out/share/icons
+  cp -r ./Fuchsia-Pop* $out/share/icons
+  runHook postInstall
+  '';
```
Comment (Zaechus) [inline pkgs/desktops/cursor/fuchsia-pop/default.nix:24]:
```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/share/icons
    cp -r ./Fuchsia-Pop* $out/share/icons
    runHook postInstall
  '';
```
Formatting

File: pkgs/desktops/cursor/fuchsia-pop/default.nix [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl }:
+
+
+stdenv.mkDerivation rec {
+  pname = "fuchsia-pop";
+  version = "2.0.0";
+
+  src =  fetchurl {
```
Comment (Zaechus) [inline pkgs/desktops/cursor/fuchsia-pop/default.nix:8]: Looks like the repo has instructions to build from source. Maybe try that instead?

**Review by Zaechus — CHANGES_REQUESTED** [review]
Also please change the PR title to reflect the [contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).

File: pkgs/top-level/all-packages.nix (hunk 3 above)
Comment (Shawn8901) [reply pkgs/top-level/all-packages.nix:39562]:
```suggestion
  fuchsia-pop = callPackage ../cursor/fuchsia-pop { };
```

doesnt it also need the `{ };`?

Comment (Zaechus) [reply pkgs/top-level/all-packages.nix:39562]: You're right I missed that.

**Pull request closed without merging**